scrapy/http/request/__init__.py

```diff
@@ -16,6 +16,9 @@
 
 class Request(object_ref):
 
+    replaced_attributes = ['url', 'method', 'headers', 'body', 'cookies', 'meta', 'flags',
```

Comment:
Considering the context I'd go for `replaceable_attributes` name and a comment explaining this are the attribute list that get propagated on `.replace()` call.

Reply:
Also, what do you think about making this list either private (`_replacable_attributes`), or upper-cased, or both?